docs-site/docs/development/development_processes/pull_request_workflow.md

```diff
@@ -0,0 +1,84 @@
+1. Changes do not degrade performance:
+   * Avoid repeated polling of endpoints that require a significant amount of overhead.
+   * Check for N + 1 queries via the SQL log.
+   * Avoid repeated access of the file system.
+1. If the pull request adds any new libraries (like gems or JavaScript libraries), they should conform to our Licensing guidelines. Also, make the reviewer aware of the new library and explain why you need it.
+
+## Definition of done
+
+If you contribute to IRIDA Next, please know that changes involve more than just code. We use the following [definition of done](https://www.agilealliance.org/glossary/definition-of-done).
+
+If a regression occurs, we prefer you revert the change. Your contribution is *incomplete* until you have made sure that it meets all these requirements.
+
+### Functionality
+
+1. Working and clean code that is commented where needed.
+1. Documented in the `/docs` directory.
+1. If your pull request adds one or more migrations, make sure to execute all migrations on a fresh database before the PR is reviewed. If the review leads to large changes in the PR, execute the migrations again after the review is complete.
+1. If your pull request adds new validations to existing models, to make sure the data processing is backwards compatible:
+   * Ask a IRIDA team member for assistance to execute the database query that checks the existing rows to ensure existing rows aren't impacted by the change.
+
+### Testing
+
+1. Unit, integration, and system tests that all pass on the CI server.
+1. Regressions and bugs are covered with tests that reduce the risk of the issue happening again.
+1. If your merge request adds one or more migrations, write tests for more complex migrations.
+
+### UI changes
+
+1. Use available components from the IRIDA Next Design System, Viral.
+1. The PR must include *Before* and *After* screenshots if UI changes are made.
+1. If the PR changes CSS classes, please include the list of affects pages, which can be found by running `grep css-class ./app -R`.
+
+### Description of changes
+
+1. Clear title and description explaining the relevancy of the contribution.
+1. Description includes any steps or setup required to ensure reviewers can view the changes you've made.
+
+### Approval
+
+1. The [PR acceptance checklist](./code_review#acceptance-checklist) has been checked as confirmed in the PR.
+1. Reviewed by relevant reviewers, and all concerns are addressed for Availability, Regressions, and Security. Documentation reviews should take place as soon as possible, but they should not block a merge request.
+1. You pull request has at least 1 approval, but depending on your changes might need additional approvals.
+  * You don't have to select any specific reviewers, but you can if you really want specific people to approve your pull request.
+1. Merged by a project maintainer.
```

Comment:
All the other items are numbered correctly when viewing on the docs site but for some reason this last one shows up with a `1.`

Reply:
Addressed in https://github.com/phac-nml/irida-next/pull/105/commits/aa95bea4d612c311ca86792f706dd567be19251c